backend/eventloop/workspace_event_loop_test.go

```diff
@@ -776,7 +776,9 @@ var _ = Describe("Workspace Event Loop Test", Ordered, func() {
 						DBRelationType:       db.APICRToDatabaseMapping_DBRelationType_SyncOperation,
 						DBRelationKey:        syncOperationRow.SyncOperation_id,
 					}
-					Expect(dbQueries.CreateAPICRToDatabaseMapping(ctx, &apiCRToDBMapping))
+
+					err = dbQueries.CreateAPICRToDatabaseMapping(ctx, &apiCRToDBMapping)
+					Expect(err).ToNot(HaveOccurred())
```

Comment:
Maybe:
```suggestion
					Expect(dbQueries.CreateAPICRToDatabaseMapping(ctx, &apiCRToDBMapping)).Should(Succeed())
```
will be more readable?

This is actually a general comment, but for existing code I wouldn't bother to fix it. But for new code, where a function only returns error, I think it's a better option.

Reply:
Yes, I understand your point. It does seem like a better approach for a function which only returns error, thank you

Reply:
Yup, agreed @nunnatsa. Speaking for myself, in unit test code I've written recently I've been using that format in preference over `err =  ; Expect(err`), and it's definitely somewhat more readable. 

But as you said, for existing code, and for this PR, I think we can leave it as is (at least for now), sound good @Rizwana777?

Reply:
yes @jgwest thank you